great_expectations/core/expectation_configuration.py

```diff
@@ -724,6 +724,15 @@ class ExpectationConfiguration(SerializableDictDot):
                 "catch_exceptions": False,
             },
         },
+        "expect_multicolumn_sum_to_equal": {
+            "domain_kwargs": ["column_list"],
+            "success_kwargs": ["sum_total"],
+            "default_kwarg_values": {
+                "result_format": "BASIC",
+                "include_config": True,
+                "catch_exceptions": False,
+            },
+        },
```

Comment:
This reminds me that I think we should add another JIRA ticket for the eventual v2 api deprecation that removes the codepath relying on this method of identifying expectation kwargs, since in all v3 api expectations, the kwargs should be drawn from the expectation implementation instead.

Reply:
@jcampbell This has been captured in JIRA: `https://superconductive.atlassian.net/browse/GE-445`.